scripts/extensions/videoEditor/src/VideoEditor.tsx

```diff
@@ -39,6 +39,7 @@ interface IState {
     cropEnabled: boolean;
     playing: boolean;
     prevVideoHeight: number;
+    etag: string;
```

Comment:
It's prone to errors if you store `etag` separately from the entity. Can you store it inside the entity object?

If it's for an article, it should be `article._etag` or if for a video project - `videoProject._etag`.

Reply:
> Can you store it inside the entity object?

Could you clarify more? I removed [article from state](https://github.com/superdesk/superdesk-client-core/pull/3218/commits/c99290faf9443cd301c8b4773d7b7eaf5000baf2) because we only need `renditions` so I'm not sure where to put it, maybe move it out of `state` or to parent?

Reply:
@idsvn I think Tomas said that `_etag` is a part of an article and it should be stored **in** the `article` itself

Reply:
Got it, I will move `renditions` and `etag` to `article` state